src/Sulu/Bundle/ContentBundle/Resources/public/js/main.js

```diff
@@ -37,6 +39,8 @@ define([
 
             'use strict';
 
+            SeoTab.initialize(app);
```

Comment:
As far as I can see [here](https://github.com/sulu/sulu/blob/c1809cddf2264af2c6ac48e1d539cf1a3507b46b/src/Sulu/Bundle/AdminBundle/Resources/public/js/main.js#L131) this file is called before `app.start` is called.